index.html

```diff
@@ -38,6 +34,149 @@ <h5 class="btn-connect">LET’S CONNECT</h5>
             </ul>
         </section>
     </header>
+    <!-- cord section start from here -->
+    <section class="card-section">
+        <div class="card">
+            <div class="card-heading">
+                <img src="images/card.png" alt="card">
+            </div>
+            <div class="card-body">
+                <h1 class="card-title">Tonic</h1>
+                <div class="card-author">
+                    <h5 class="heading5">CANOPY</h5>
+                    <div class="dot"></div>
+                    <p>Back End Dev</p>
+                    <div class="dot"></div>
+                    <p>2015</p>
```

Comment:
- [ ] I would like to recommend you use a list `ul` instead of **div** and **p** tag It will be better for semantics reasons :smile: